### PR TITLE
List down active proposals in the Ballerina dev website community page

### PR DESCRIPTION
**Pull request by bimsarapathiraja**
## Purpose
Listing down active proposals for GitHub repos in Ballerina platform in the community page of the Ballerina website.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: .github/workflows/generatemd.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+name: Generate Proposal MD
+
+on:
+  workflow_dispatch: #periodic
```
Comment (keizer619) [inline .github/workflows/generatemd.yml:4]:
```suggestion
  workflow_dispatch:
```

File: .github/workflows/generatemd.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    inputs:
+      logLevel:
```
Comment (keizer619) [inline .github/workflows/generatemd.yml:6]: @bimsarapathiraja This input is not used right? Shall we remove in that case?

File: .github/workflows/generatemd.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        description: "Log lev"
+        required: true
+        default: "warning"
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  generateHTML:
+    env:
+      JBAL_VERSION: "2201.0.3"
```
Comment (keizer619) [inline .github/workflows/generatemd.yml:16]: @bimsarapathiraja Do we need this as we use Ballerina GitHub Action?

File: .github/workflows/generatemd.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      - name: Commit files and Push
+        id: commit
+        run: |
+          DIFF_VAR=false
+          if git diff --quiet HEAD --; then
+              echo "No changes"             
+          else
+              DIFF_VAR=true
+              UUID_VAL=$(uuidgen)
+              echo $UUID_VAL
```
Comment (keizer619) [inline .github/workflows/generatemd.yml:40]: @bimsarapathiraja Do we need this echo? is it for testing purpose? If so lets remove this

File: .github/workflows/generatemd.yml (hunk 3 above)
Comment (bimsarapathiraja) [reply .github/workflows/generatemd.yml:11]: env part?

File: .github/workflows/generatemd.yml (hunk 4 above)
Comment (bimsarapathiraja) [reply .github/workflows/generatemd.yml]: Removed

**Review by praneesha — APPROVED** [review]

File: .github/workflows/generatemd.yml (hunk 2 above)
Comment (bimsarapathiraja) [reply .github/workflows/generatemd.yml]: removed

File: .github/workflows/generatemd.yml (hunk 3 above)
Comment (bimsarapathiraja) [reply .github/workflows/generatemd.yml]: removed

**Review by keizer619 — APPROVED** [review]

File: pages/community/active-proposals.js [hunk 5]
```diff
@@ -0,0 +1,354 @@
+import fs from "fs";
+import matter from "gray-matter";
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import { Container, Col, Button, Offcanvas } from "react-bootstrap";
```
Comment (sm1990) [inline pages/community/active-proposals.js:5]: Remove Button & Offcanvas as they are not used anymore.

File: pages/community/active-proposals.js [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+import MarkdownNavbar from "markdown-navbar";
```
Comment (sm1990) [inline pages/community/active-proposals.js:6]: Remove MarkdownNavBar as we do not use it.

File: pages/community/active-proposals.js [hunk 7]
```diff
@@ -0,0 +1,354 @@
...
+import Image from "next-image-export-optimizer";
+import rehypeRaw from "rehype-raw";
+import Head from "next/head";
+
+import { getHighlighter, setCDN } from "shiki";
+
+setCDN("https://unpkg.com/shiki/");
+
+import Layout from "../../layouts/LayoutOther";
+//import LeftNav from "../../components/common/left-nav/LeftNav";
```
Comment (sm1990) [inline pages/community/active-proposals.js:18]: Remove comments.

Comment (bimsarapathiraja) [reply pages/community/active-proposals.js:18]: All comments?

**Review by sm1990 — APPROVED** [review]

**Review by keizer619 — CHANGES_REQUESTED** [review]
package-lock.json has 8185 line deletions and 62 line additions

File: .github/scripts/bal_proposals_bot/target/build [hunk 8]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (keizer619) [inline .github/scripts/bal_proposals_bot/target/build:1]: @bimsarapathiraja Shall we put `target` into ignore list and remove the files committed in `target`

**Review by keizer619 — APPROVED** [review]

**Pull request merged**